src/logic/calculate.js

```diff
@@ -0,0 +1,106 @@
+import operate from './operate';
+
+function isNumber(item) {
+  return !!item.match(/[0-9]+/);
+}
+
+export default function calculate(obj, buttonName) {
+  if (buttonName === 'AC') {
+    return {
+      total: 0,
+      next: null,
+      operation: null,
+    };
+  }
```

Comment:
- [ ] It is good to display a default zero on the screen when you open an application and after performing an operation when you press `AC` I recommend to display a zero in the screen as well